README.md

```diff
@@ -28,15 +28,34 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. Describe the biggest difference between `.forEach` & `.map`.
+1. Describe the biggest difference between `.forEach` & `.map`.  
 
-2. What is the difference between a function and a method?
 
-3. What is closure?
+forEach: This iterates over a list and applies some operation to each list member .
+map: This iterates over a list, transforms each member of that list, and returns another list of the same size with the transformed members 
 
-4. Describe the four rules of the 'this' keyword.
 
-5. Why do we need super() in an extended class?
+
+2. What is the difference between a function and a method?  
+
+'method' is the object-oriented word for 'function'. That's pretty much all there is to say.
+
+3. What is closure?   
+
+a closure gives you access to an outer function’s scope from an inner function. 
+
+4. Describe the four rules of the 'this' keyword. 
```

Comment:
Notice this question is asking you to describe or explain the four rules, not just list them. Good job on knowing them, though.